crates/db_views/src/post_view.rs

```diff
@@ -607,8 +611,11 @@ impl PaginationCursor {
 #[derive(Clone)]
 pub struct PaginationCursorData(PostAggregates);
 
-#[derive(Default, Clone)]
+#[derive(Clone, TypedBuilder)]
+#[builder(field_defaults(default))]
 pub struct PostQuery<'a> {
+  #[builder(!default)]
+  pub local_site: LocalSite,
```

Comment:
Need to change this to builder to ensure that LocalSite gets passed in. Using Default is not a good idea because LocalSite itself implements Default, so it would be very easy to forget the parameter.

Reply:
LocalSite can instead be added to the list function's parameters, which is how non-optional PersonId is done for other views if i remember correctly

Reply:
Youre right, I could figure this out but now I got it. Wasted a lot of time rewriting all the code and tests to use builder.